### PR TITLE
Update webpack

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 const path = require('path');
+const webpack = require('webpack');
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
+  mode: 'development',
```
Comment (nealstewart) [inline config/webpack.dev.js]: does the mode passed by the command line override this?

File: config/webpack.prod.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 const path = require('path');
-const UglifyJsPlugin = require('webpack/lib/optimize/UglifyJsPlugin');
+const webpack = require('webpack');
+
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 module.exports = {
   devtool: 'cheap-source-map',
+  mode: 'production',
```
Comment (nealstewart) [inline config/webpack.prod.js]: ignore previous comment

File: config/webpack.dev.js (hunk 1 above)
Comment (thomasthiebaud) [reply config/webpack.dev.js]: I'm not passing the mode from the command line anymore

File: config/webpack.prod.js (hunk 2 above)
Comment (thomasthiebaud) [reply config/webpack.prod.js]: I'm not passing the mode from the command line anymore ;)

**Comment by nealstewart** [comment]
this seems to decrease the built size! all for it :)

**Pull request merged**